src/librustdoc/html/static/js/search.js

```diff
@@ -2004,6 +2016,8 @@ function initSearch(rawSearchIndex) {
             const itemParentIdxs = crateCorpus.i;
             // an array of (Object | null) the type of the function, if any
             const itemFunctionSearchTypes = crateCorpus.f;
+            // an array of (boolean) whether the item is deprecated
```

Comment:
It looks like this is actually an array of `String | null` with the deprecation message. (It'd be nice to mention in the commit/PR that the message is also being shown in search now too, and have a screenshot of what that looks like).

Reply:
Ah yeah I had it as boolean initially and missed this bit, good catch